charts/studio/templates/configmap-studio-datachain-worker.yaml

```diff
@@ -3,6 +3,9 @@ kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-datachain-worker
 data:
+  {{- if and (or .Values.studioBackend.replicaCount .Values.studioBackend.autoscaling.enabled) (or .Values.studioDatachainWorker.replicaCount .Values.studioDatachainWorker.autoscaling.enabled) }} 
```

Comment:
Now we don't break existing uses of the Helm chart:

* If both `studio-backend` and `studio-datachain-worker` are deployed to the same cluster, establish the connection through `INTERNAL_API_URL`
* Else, establish the connection through `API_URL`